web-app/src/pages/tournament/TournamentCreator.vue

```diff
@@ -191,18 +191,22 @@ export default class TournamentCreator extends Vue {
       align: 'left',
       field: 'name',
       sortable: true,
+      classes: 'textWrapDotted',
```

Comment:
`classes` name refer to array of class names/strings, not to for single/standard string